db/migrate/20160726061957_turn_default_value_to_lookup_value.rb

```diff
@@ -0,0 +1,26 @@
+class TurnDefaultValueToLookupValue < ActiveRecord::Migration
+
+  def up
+    LookupValue.transaction do
+      LookupKey.unscoped.each do |key|
+        LookupValue.create(:match => 'default', :value => key.default_value, :lookup_key_id => key.id, :use_puppet_default => key.use_puppet_default)
+      end
+    end
+
+    remove_column :lookup_keys, :default_value
+    remove_column :lookup_keys, :use_puppet_default
+  end
+
+  def down
+    add_column :lookup_keys, :default_value, :string, :limit => 255
+    add_column :lookup_keys, :use_puppet_default, :boolean
+
+    default_lookup_values = LookupValue.where(:match => 'default')
+
+    default_lookup_values.each do |lv|
```

Comment:
`find_each` instead of `each` will run this in batches which could save memory and time on instances with >1k lookup keys.